src/core/Notification/InlineNotification/InlineNotification.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { Component, forwardRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlDivWithRef } from '../../../reset';
+import { Icon } from '../../Icon/Icon';
+import { Heading } from '../../Heading/Heading';
+
+import {
+  BaseNotificationProps,
+  notificationClassNames,
+  baseClassName,
+} from '../BaseNotification/BaseNotification';
+import { AutoId } from '../../utils/AutoId/AutoId';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { baseStyles } from './InlineNotification.baseStyles';
+
+export interface InlineNotificationProps extends BaseNotificationProps {
```

Comment:
The InlineNotification name is still misleading in my opinion, since it does not exist for inline use.

Reply:
Was there discussion that there is only `Notification` component because there was only one prop or so that was different? We have to discuss this.

Reply:
Not a proper one yet. The styles are somewhat different with the success notification not having buttons, but it might be doable still. Would be much cleaner. I think this solution came from using a copy+paste of Alert as a base for the component.

Reply:
Inline removed